src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,43 @@
+import { FC, PropsWithChildren } from "react"
+
+import clsxm from "@/lib/clsxm"
+
+import { FortIconClose } from "@/icons"
+
+export type ToastComponentProps = {
+  isVisible: boolean
+  onDismiss?: () => void
+  className?: string
+}
+
+export const Toast: FC<PropsWithChildren<ToastComponentProps>> = ({
+  className,
+  isVisible,
+  onDismiss,
+  children,
+}) => {
+  return (
+    <div
+      className={clsxm(
+        `${
+          isVisible ? "animate-enter opacity-100" : "animate-leave opacity-0"
```

Comment:
Couple things here:

1. If you need to do conditional classes, you should use `clsxm`:

```tsx
<div
  className={
    (clsxm(
      "w-full max-w-xs rounded-md bg-gray-800 p-4 text-gray-400 shadow",
      {
        "animate-enter opacity-100": isVisible,
        "animate-leave opacity-0": !isVisible,
      }
    ),
    className)
  }
>
```

2. But... you should use `<Transition>` to animate this